rust/agama-lib/src/lib.rs

```diff
@@ -41,13 +41,34 @@ mod store;
 pub use store::Store;
 pub mod questions;
 use crate::error::ServiceError;
+use reqwest::cookie::Jar;
+use reqwest::{header, Client};
+use std::sync::Arc;
 
 const ADDRESS: &str = "unix:path=/run/agama/bus";
 
 pub async fn connection() -> Result<zbus::Connection, ServiceError> {
     connection_to(ADDRESS).await
 }
 
+pub fn http_client(token: String) -> Result<reqwest::Client, ServiceError> {
+    let cookie_store = Arc::new(Jar::default());
```

Comment:
Are you using the cookie store for anything? If not, please, disable it.

Reply:
No, we are not using it, but as commented in the description we could save the cookies to a file and load it when obtaining the client, but as for this PBI the focus was local access we could omit it by now.